templates/base.html

```diff
@@ -0,0 +1,34 @@
+{% load static %}
+
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="{% static 'css/bootstrap-theme.min.css' %}" />
+        <link rel="stylesheet" type="text/css" href="{% static 'css/bootstrap.css' %}" />
+    </head>
+<body>
+
+<nav class="navbar navbar-default">
```

Comment:
why the F did you put a navbar? it is not part of your task. ALSO the homepage is not the playlist page